style/variables.css

```diff
@@ -4,38 +4,43 @@
 |----------------------------------------------------------------------------*/
 
 :root {
-  --jp-icon-discard-file-selected: url('./images/discard-selected.svg');
-  --jp-icon-move-file-down-hover: url('./images/move-file-down-hover.svg');
-  --jp-icon-move-file-up-hover: url('./images/move-file-up-hover.svg');
   --jp-checkmark: url('./images/checkmark-icon.svg');
-  --jp-icon-clear-white: url('./images/clear-white.svg');
-  --jp-icon-git-clone: url('./images/git-clone-icon.svg');
-  --jp-git-diff-deleted-color: rgba(255, 0, 0, 0.2);
   --jp-git-diff-added-color: rgba(155, 185, 85, 0.2);
-  --jp-git-diff-output-color: rgba(0, 141, 255, 0.3);
+  --jp-git-diff-deleted-color: rgba(255, 0, 0, 0.2);
   --jp-git-diff-output-border-color: rgba(0, 141, 255, 0.7);
+  --jp-git-diff-output-color: rgba(0, 141, 255, 0.3);
+  --jp-icon-clear-white: url('./images/clear-white.svg');
+  --jp-icon-discard-file-selected: url('./images/discard-selected.svg');
+  --jp-icon-git-branch-light-theme: url('./images/git-branch.svg');
```

Comment:
get rid of this var once you've fixed up the dark theme stuff for the new branch dialog box

Reply:
Removed.